cmd/util/cmd/execution-state-extract/execution_state_extract_test.go

```diff
@@ -136,12 +134,19 @@ func TestExtractExecutionState(t *testing.T) {
 				//we need fresh output dir to prevent contamination
 				unittest.RunWithTempDir(t, func(outdir string) {
 
-					Cmd.SetArgs([]string{"--execution-state-dir", execdir, "--output-dir", outdir, "--block-hash", blockID.String(), "--datadir", datadir, "--no-migration", "--no-report"})
+					Cmd.SetArgs([]string{
+						"--execution-state-dir", execdir,
+						"--output-dir", outdir,
+						"--state-commitment", stateCommitment.String(),
+						"--datadir", datadir,
+						"--no-migration",
+						"--no-report",
+						"--chain", flow.Emulator.Chain().String()})
 
 					err := Cmd.Execute()
 					require.NoError(t, err)
 
-					require.FileExists(t, path.Join(outdir, bootstrap.FilenameWALRootCheckpoint)) //make sure we have root checkpoint file
+					// require.FileExists(t, path.Join(outdir, bootstrap.FilenameWALRootCheckpoint)) //make sure we have root checkpoint file
```

Comment:
@vishalchangrani do you remember why we commented this out? In the commit it says it was to fix this test, does that mean we don't want to bother checking if this file exists here? if so we should just delete this line

Reply:
yeah - i believe this line is no longer needed.

Reply:
```suggestion
```

Reply:
if we're going to do another round of adjustments, could we remove this line @ramtinms